packages/manager-react-components/src/components/datagrid/datagrid-cursor.stories.tsx

```diff
@@ -44,6 +44,7 @@ const DatagridStory = (args) => {
 
   return (
     <Datagrid
+      {...args}
```

Comment:
I would recommend destructuring of args in parameters (L32) to make it more clear : `{items, ...datagridArgs}`

Reply:
indeed, done !